libs/stats/odc/stats/_pq.py

```diff
@@ -33,7 +36,7 @@ class StatsPQ(StatsPluginInterface):
 
     def __init__(
         self,
-        filters: Optional[List[Iterable[Tuple[str, int]]]] = None,
+        filters: Optional[List[Dict[str, Iterable[Tuple[str, int]]]]] = None,
```

Comment:
```suggestion
        filters: Optional[Dict[str, Iterable[Tuple[str, int]]]] = None,
```
no need for extra list